### PR TITLE
LPS-178147 

### PR DESCRIPTION
**Pull request by locpham97**
Hi @ces-quanhuynh ,
Please help me review this ticket
https://issues.liferay.com/browse/LPS-178147 
Thanks,
Loc

### CODE REVIEW

**Review by ces-quanhuynh — COMMENTED** [review]
Hi @locpham97,
Please check again.
Thanks,
Quan

File: modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/scheduler/helper/AssetEntriesCheckerHelper.java [hunk 1]
```diff
@@ -309,8 +309,8 @@ private List<AssetEntry> _getAssetListEntrySelectedAssetEntries(
 			catch (PortalException portalException) {
 				if (_log.isWarnEnabled()) {
 					_log.warn(
-						"Unable to get segments entry IDs for " +
-							"asset list entry " + assetListEntryId);
+						"Unable to get segments entry IDs for asset list " +
+							"entry " + assetListEntryId);
```
[the remark this answers is not in the document]
Comment (locpham97) [reply modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/scheduler/helper/AssetEntriesCheckerHelper.java:313]: Hey @ces-quanhuynh , This is just SF.

File: modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/internal/display/context/AssetCategoriesNavigationDisplayContext.java [hunk 2]
```diff
@@ -216,7 +215,7 @@ private String _getPortletURL(long categoryId) {
 			portletURL.setParameter("categoryId", String.valueOf(categoryId));
 		}
 
-		return HtmlUtil.escape(portletURL.toString());
+		return portletURL.toString();
```
[the remark this answers is not in the document]
Comment (locpham97) [reply modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/internal/display/context/AssetCategoriesNavigationDisplayContext.java:218]: The categoryId in the URL category filter when user pagination is escaped. As a result, the portlet cannot get the categoryId.

**Comment by ces-quanhuynh** [comment]
Forward https://github.com/liferay-lima/liferay-portal/pull/4218

**Comment by ces-quanhuynh** [comment]
Forward https://github.com/liferay-echo/liferay-portal/pull/11763
cc @locpham97

**Pull request closed without merging**